Ports/cowsay/package.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='cowsay'
+version='3.04'
+depends=('perl5')
+useconfigure='false'
+files="https://github.com/tnalpgge/rank-amateur-cowsay/archive/refs/tags/cowsay-${version}.tar.gz cowsay-${version}.tar.gz d8b871332cfc1f0b6c16832ecca413ca0ac14d58626491a6733829e3d655878b"
+workdir="rank-amateur-cowsay-cowsay-${version}/"
+
+prefix='/usr/local'
+bin_path="${prefix}/bin"
+cow_path="${prefix}/share/cows"
+perl_executable="${bin_path}/perl"
+
+build() {
+    run sed -i "s#%PREFIX%#${prefix}#" cowsay
+    run sed -i "s#%BANGPERL%#!${perl_executable}#" cowsay
+}
+
+install() {
+    run mkdir -p "${SERENITY_INSTALL_ROOT}/${bin_path}/"
+    run cp cowsay "${SERENITY_INSTALL_ROOT}/${bin_path}/"
+    run mkdir -p "${SERENITY_INSTALL_ROOT}/${cow_path}/"
+    run cp -r cows/ "${SERENITY_INSTALL_ROOT}/${cow_path}/"
+    run ln -sf "${bin_path}/cowsay" "${SERENITY_INSTALL_ROOT}/${bin_path}/cowthink"
```

Comment:
I think both `mkdir`s and the `ln` actions can be run with `run_nocd`, since the paths are all absolute anyway.